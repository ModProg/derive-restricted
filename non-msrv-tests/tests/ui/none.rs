#[derive(derive_where::DeriveWhere)]
struct Test<T>(T);

fn main() {}
