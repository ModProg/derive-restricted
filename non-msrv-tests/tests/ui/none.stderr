error: no traits found to implement, use `#[derive_where(..)` to specify some
 --> tests/ui/none.rs:2:1
  |
2 | struct Test<T>(T);
  | ^^^^^^^^^^^^^^^^^^
